src/trace/dma-trace.c

```diff
@@ -335,6 +338,11 @@ int dma_trace_enable(struct dma_trace_data *d)
 	d->enabled = 1;
 	schedule_task(&d->dmat_work, DMA_TRACE_PERIOD, DMA_TRACE_PERIOD);
 
+	/* it should be the very first sent log for easily identification */
+	tr_info(&dt_tr, "FW ABI 0x%x DBG ABI 0x%x tag " SOF_GIT_TAG " src hash 0x%x (ldc hash " META_QUOTE(SOF_SRC_HASH) ")",
+		SOF_ABI_VERSION, SOF_ABI_DBG_VERSION, SOF_SRC_HASH);
```

Comment:
@ktrzcinx do you remember why you put _both_ `META_QUOTE(SOF_SRC_HASH)` _and_ `%x SOF_SRC_HASH` here? Do they ever look different from each other?

Reply:
They differ in case of using incompatible `.ldc` file, it's because `META_QUOTE(SOF_SRC_HASH)` comes from used `.ldc` file and `SOF_SRC_HASH` is .ri buildin variable.

Reply:
Thanks @ktrzcinx !

> Do they ever look different from each other?

Answering that part of my own question: they do but only when forcing `sof-logger` to use an incompatible `.ldc` file with the `-n` option.